be/src/service/doris_main.cpp

```diff
@@ -275,6 +276,9 @@ int main(int argc, char** argv) {
 #if defined(LEAK_SANITIZER)
         __lsan_do_leak_check();
 #endif
+        LOG(INFO) << "Tcmalloc New number: " << new_hook_calls
```

Comment:
Better not print log here, use metrics

Reply:
This is temp log, forgot to delete it  =_=